src/test/java/seedu/ddd/model/contact/common/predicate/ClientTypePredicateTest.java

```diff
@@ -3,8 +3,8 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.ddd.testutil.TypicalContacts.ALICE;
-import static seedu.ddd.testutil.TypicalContacts.BOB;
+import static seedu.ddd.testutil.contact.TypicalContacts.ALICE;
```

Comment:
If my BOB changed to VALID_VENDOR, should ALICE be changed to a VALID_CLIENT?

Reply:
`ALICE` belongs to the list returned by `getTypicalContacts`, so I left her unchanged. 

However, I did change `AMY` into `VALID_CLIENT` for clarity:

- `AMY` -> `VALID_CLIENT`
- `BOB` -> `VALID_VENDOR`